src/wp-includes/general-template.php

```diff
@@ -63,9 +64,10 @@ function get_header( $name = null, $args = array() ) {
  * @since 5.5.0 A return value was added.
  * @since 5.5.0 The `$args` parameter was added.
  *
- * @param string $name The name of the specialized footer.
- * @param array  $args Optional. Additional arguments passed to the footer template.
- *                     Default empty array.
+ * @param string|null $name The name of the specialized footer.
+ *                          Default null.
```

Comment:
```suggestion
 * @param string|null $name The name of the specialized footer. Default null.
```

As it's small text use single line

Reply:
@mukeshpanchal27 Thank you for review! I've fixed it with the latest commit